Cargo.lock

```diff
@@ -146,9 +156,9 @@ dependencies = [
 
 [[package]]
 name = "anstream"
-version = "0.6.5"
+version = "0.6.7"
```

Comment:
I am not sure if it's okay to edit it manually since there's a warning at the beginning of Cargo.lock
# This file is automatically @generated by Cargo.
# It is not intended for manual editing.